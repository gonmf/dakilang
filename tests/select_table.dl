
other(X) :- abc(X, "gual").
other(X) :- abc(X, "guali").
other(X) :- abc(X, "gual").

other(X) :- abc(X, "guali"); abc(X, "gual").

listing

select_table 9

listing

select_table 0

listing

select_table




