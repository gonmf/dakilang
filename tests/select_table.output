>
> other(X) :- abc(X, "gual").
> other(X) :- abc(X, "guali").
> other(X) :- abc(X, "gual"). # This instruction will produce a warning
Clause already exists

>
> other(X) :- abc(X, "guali"); abc(X, "gual").
>
> listing
0: other(X) :- abc(X, 'gual').
1: other(X) :- abc(X, 'guali').

>
> select_table 9
Table changed to 9

>
> listing

>
> select_table 0
Table changed to 0

>
> listing
0: other(X) :- abc(X, 'gual').
1: other(X) :- abc(X, 'guali').

>
> select_table
Current table is 0

>
>
>
>
