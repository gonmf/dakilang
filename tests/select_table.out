>
> other(X) :- abc(X, "gual").
> other(X) :- abc(X, "guali").
> other(X) :- abc(X, "gual").
>
> other(X) :- abc(X, "guali"); abc(X, "gual").
>
> listing
0: other(X) :- abc(X, 'gual'). (3)
1: other(X) :- abc(X, 'guali'). (2)

>
> select_table 9
Table changed to 9

>
> listing

>
> select_table 0
Table changed to 0

>
> listing
0: other(X) :- abc(X, 'gual'). (3)
1: other(X) :- abc(X, 'guali'). (2)

>
> select_table
Current table is 0

>
>
>
>
